<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateWorkOrderWidget</class>
 <widget class="QWidget" name="CreateWorkOrderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Work Order</string>
  </property>
  <widget class="QPushButton" name="createNewWOSaveButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>440</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Work Order</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Shipping Method</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>212</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="createWODeliveryAoFRadioButton">
       <property name="text">
        <string>Delivery (Address on File)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="createWODeliveryOtherRadioButton">
       <property name="text">
        <string>Delivery (Other)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="createWODeliveryPickupRadioButton">
       <property name="text">
        <string>Customer Pickup</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="createNewWOBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Create New Work Order</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Work Order Number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextBrowser" name="CreateWorkOrderNumberField">
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>25</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:9.75pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic; color:#888888;&quot;&gt;Auto-generated upon completion&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Customer</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="createWOCustomerSelection">
       <item>
        <property name="text">
         <string>Select...</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Order Date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="createWODateInput"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Quantity</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="createWOQuantityInput"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Production Date</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="createWOProductionDateInput"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="requiredByDateLabel">
       <property name="text">
        <string>Required By: </string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="createWORequiredByDate"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="productTemplateViewButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Template</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
